implementations/rust/ockam/ockam_command/src/node/create/config.rs

```diff
@@ -218,9 +218,31 @@ impl NodeConfig {
         identity_name: &String,
     ) -> miette::Result<()> {
         debug!("Running node config");
-        for section in self.parse_commands(node_name, identity_name)? {
+
+        // Run `project enroll` first
+        if let Some(cmd) = self
+            .project_enroll
+            .clone()
+            .into_parsed_commands(Some(identity_name))?
+            .into_iter()
+            .next()
+        {
+            cmd.run(ctx, opts).await?;
+
+            // Newline before the `node create` command
+            opts.terminal.write_line("")?;
+            debug!("project enroll command finished");
+
+            // Unset the `ENROLLMENT_TICKET` env var, so that the `node create` command
+            // doesn't try to use it again
+            std::env::remove_var("ENROLLMENT_TICKET");
```

Comment:
Could you please move it to a const in one of your PRs in the future?

Reply:
Yup, I'm preparing another pr to clean this up and add a proper test for this.